lib/Dancer2/Plugin/Swagger2.pm

```diff
@@ -224,8 +224,20 @@ sub _validate_request {
                 next;
             }
 
+
             my $value  = $values[0];
-            my %input  = ( $name => $value );
+
+            # TODO steal more from Mojolicious::Plugin::Swagger2 ;-)
+            if ($type and defined ($value //= $parameter_spec->{default})) {
```

Comment:
We are validating header, query, path, and formData in this part of the code. The $value is only undef if the parameter does not exist at all. Otherwise it will be the empty string.
Can you show me an example where we explicitly pass an undef value via header, query, path or formData?